### PR TITLE
CRDS-388 New NIRISS SOSS extract_1d reference files

### PR DESCRIPTION
**Pull request by stscieisenhamer**
Resolves CCD-[967](https://jira.stsci.edu/browse/CCD-967)/[968](https://jira.stsci.edu/browse/CCD-968)/[969](https://jira.stsci.edu/browse/CCD-969)/[970](https://jira.stsci.edu/browse/CCD-970)
WIP [CRDS-388](https://jira.stsci.edu/browse/CRDS-388) and all subtasks

### CODE REVIEW

File: crds/jwst/specs/combined_specs.json [hunk 1]
```diff
@@ -5131,6 +5131,100 @@
             "tpn":"niriss_saturation.tpn",
             "unique_rowkeys":null
         },
+        "speckernel":{
```
Comment (eslavich) [inline crds/jwst/specs/combined_specs.json:5134]: Sounds like some kind of rare songbird

File: crds/jwst/specs/niriss_specprofile.rmap [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    'filekind' : 'SPECPROFILE',
+    'filetype' : 'SPECPROFILE',
+    'instrument' : 'NIRISS',
+    'mapping' : 'REFERENCE',
+    'name' : 'jwst_niriss_specprofile.rmap',
+    'observatory' : 'JWST',
+    'parkey' : (('META.EXPOSURE.TYPE', 'META.SUBARRAY.NAME', 'META.INSTRUMENT.FILTER'), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
+    'sha1sum' : '16e6119fb13458c8fbda49f0d03905707e6c7be4',
+    'suffix' : 'specprofile',
+    'text_descr' : '  GR700XD Grism Trace Profile Mapping File',
```
Comment (eslavich) [inline crds/jwst/specs/niriss_specprofile.rmap:14]: Extra leading spaces here

File: crds/jwst/specs/combined_specs.json (hunk 1 above)
Comment (stscieisenhamer) [reply crds/jwst/specs/combined_specs.json:5134]: Possibly some bird sang about by Paul McCartney...

**Review by eslavich — APPROVED** [review]
LGTM

**Pull request merged**